packages/bitmasks/bitmasks.1.1.0/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "david.allsopp@metastack.com"
+authors: [ "David Allsopp" ]
+license: "BSD-3-clause with OCaml linking exception"
+homepage: "https://metastack.github.io/bitmasks"
+dev-repo: "https://github.com/metastack/bitmasks.git"
+bug-reports: "https://github.com/metastack/bitmasks/issues"
+build: [ "jbuilder" "build" "@install" ]
+build-doc: [
+  ["jbuilder" "build" "@doc"]
+  ["cp" "-r" "_build/default/_doc/bitmasks" "_build/default/_doc/index.html" "_build/default/_doc/odoc.css" "%{doc}%/"]
```

Comment:
Does jbuilder not install the doc files with the ```@doc``` and ```@install``` argument or something ? I feel like there should be a cleaner solution than that but I'm not a jbuilder expert.

Reply:
poke @rgrinberg

Reply:
Jbuilder doesn't install docs, no... although I realise I've forgotten to include a corresponding `rm` for removing (though that's not needed for opam2, of course)

Reply:
Jbuilder doesn't install docs currently. Though there's a PR that talks about implementing this. I'm not even sure how it's going to work though, seems like docs need their own .install file, right?

Reply:
I just looked at this again - I dismissed an earlier attempt to sort out the doc target too quickly (that'll teach me to do these things during rehearsals...). I've adapted the opam package with a back-ported commit which updates the .install file with the odoc-generated files and sorts this out cleanly (using the .install file of course means that `opam remove` works without warning even in opam 1.2.2).

@rgrinberg - off the top of my head, I can't remember if opam processes all *.install, or just the package's one (I think it's just the package's one). There are various ways it could work - but it's probably best holding off a bit while odoc still stabilises (installing a separate html index for each package is clearly not the "final" opam, jbuilder or odoc solution, but it'll do for now!)